### ISSUE TITLE
Bug: Incorrect scroll position when setting calendarMinDateId

### ISSUE THREAD

**Issue by subramanian-elavathur** [MarceloPrado/flash-calendar#16]
### Code to reproduce

```typescript
import {Calendar, toDateId} from '@marceloterreiro/flash-calendar';
import React, {useState} from 'react';
import {Text, View} from 'react-native';

const today = toDateId(new Date());

export function BasicCalendarList() {
  const [selectedDate, setSelectedDate] = useState(today);

  return (
    <View style={{flex: 1, paddingVertical: 50}}>
      <Text>Selected date: {selectedDate}</Text>
      <Calendar.List
        calendarActiveDateRanges={[
          {
            startId: '2024-02-25',
            endId: '2024-03-02',
          },
        ]}
        calendarInitialMonthId={'2024-01-01'}
        calendarMinDateId={'2023-02-27'}
        onCalendarDayPress={setSelectedDate}
      />
    </View>
  );
}
```

### Actual Scroll Position -> based on `calendarMinDateId`

<img width="358" alt="Screenshot 2024-02-28 at 7 38 38 AM" src="https://github.com/MarceloPrado/flash-calendar/assets/4240966/67f65c4d-b184-4c62-99e9-4026e3b7be76">

### Expected scroll position -> based on `calendarInitialMonthId` should be February 2024

<img width="358" alt="Screenshot 2024-02-28 at 7 39 41 AM" src="https://github.com/MarceloPrado/flash-calendar/assets/4240966/bc27a705-16f9-4661-997c-ac69e317784e">

Please let me know if you need any other details.

And just wanted to say thanks a lot for building flash-calendar, barring this small bug and bug #11 (thanks for fixing this)  it has been a very smooth experience upgrading our apps entire calendar to use flash-calendar!

cc: @abs192

**Comment by MarceloPrado** [comment]
Fixed in 0.0.7 release.

**Comment by subramanian-elavathur** [comment]
Thank you @MarceloPrado - we confirm the issue is resolved in v0.0.7!
